src/Spectre.libGaussianMixtureModelling.Tests/ExpectationMaximizationTest.cpp

```diff
@@ -34,8 +34,8 @@ class ExpectationMaximizationTest : public ::testing::Test
 {
 protected:
 
-    std::vector<double> mzs;
-    std::vector<double> intensities;
+    std::vector<DataType> mzs;
```

Comment:
Whoa, you already defined `std::vector<DataType>` as `Data` so why aren't you using it? The same goes to all repetitions across the changed files.